docs/install_linux_mac.rst

```diff
@@ -286,7 +286,11 @@ Or, to install with PostgreSQL support:
 .. note::
 
   To install the development version, replace ``Red-DiscordBot`` in the above commands with the
-  following link:
+  link below. **The development version of the bot contains experimental changes. It is not
+  intended for normal users.** We will not support anyone using the development version in any
+  support channels. Using the develepment version may break third party cogs and not all core
+  commands may work. Downgrading after an accidental install of dev may cause data loss, crashes
```

Comment:
I don't think we should say "after an accidental install of dev", since even purposeful installs have these side effects. Maybe "Downgrading after installing the development version may..."